package.json

```diff
@@ -19,7 +19,6 @@
   "license": "ISC",
   "dependencies": {
     "@mapbox/mapbox-gl-style-spec": "^13.12.0",
-    "nan": "^2.14.0",
```

Comment:
nice catch.

Reply:
I think it's ready now :)

Reply:
Same. I'll assign @mapbox/tilesets-api for final 👍 review.